### ISSUE TITLE
train my own data

### ISSUE THREAD

**Issue by sankim90** [chuanqi305/MobileNet-SSD#158]
Hi

- My class
CLASSES = ('background',
           'person', 'bicycle', 'car', 'motorcycle',
           'bus', 'truck', 'traffic light', 'stop sign', 'dog'

- run gen_model.sh 10

- train.sh is
#!/bin/sh
if ! test -f example/MobileNetSSD_train.prototxt ;then
	echo "error: example/MobileNetSSD_train.prototxt does not exist."
	echo "please use the gen_model.sh to generate your own model."
        exit 1
fi
mkdir -p snapshot
/home/sankim/caffe-ssd/build/tools/caffe train -solver="solver_train.prototxt" \
-gpu 0 

- run demo.py
net_file= 'MobileNetSSD_deploy.prototxt'  
caffe_model='mobilenet_iter_30000.caffemodel'

The train is over but the model performs incorrect detection

Do I have to add -weight MobileNetSSD_deploy.caffemodel to train.sh?

**Comment by soufianesabiri** [comment]
Were you able to train your model? If so please show me the steps... Thanks!

**Comment by ibrahimsherdil** [comment]
@sankim90 were you able to know what was the issue?

**Comment by sankim90** [comment]
@soufianesabiri @ibrahimsherdil Nope, I did not use the MobileNet SSD because I thought the model was unreliable.
